tomviz/MainWindow.ui

```diff
@@ -184,7 +184,7 @@
         <bool>true</bool>
        </property>
        <property name="columnCount">
-        <number>2</number>
+        <number>1</number>
```

Comment:
Qt Designer updates this automatically to what it thinks is right, but this being 1 causes crashes when interacting with the OperatorsWidget.  Can you set it back to 2?